Sources/SmithyTestUtil/RequestTestUtil/HttpRequestTestBase.swift

```diff
@@ -223,22 +229,13 @@ open class HttpRequestTestBase: XCTestCase {
     public func genericAssertEqualHttpBodyData(
         expected: ByteStream,
         actual: ByteStream,
-        isXML: Bool,
-        isJSON: Bool,
-        _ callback: (Data, Data) -> Void,
+        contentType: HTTPBodyContentType,
```

Comment:
The `genericAssertEqualHttpBodyData()` method gets an enumeration for content-type of the body to be checked, and the callback is removed since it is no longer needed (it was used to check body structure members, which is unnecessary & does not yield correct results in some cases.)